delta/plugins/archive/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/archive/Archives.scala

```diff
@@ -89,7 +88,7 @@ class Archives(
       value: ArchiveValue
   )(implicit subject: Subject): IO[ArchiveRejection, ArchiveResource] =
     (for {
-      p   <- projects.fetchProject(project, notDeprecatedOrDeleted)
+      p   <- fetchContext.onRead(project)
```

Comment:
These project options were also a bit complicated when we have in the end 3 cases: read, create, modify

Reply:
Also it is now also possible to create archives for deprecated project/orgs/exceeded quotas